src/dss/purge.py

```diff
@@ -0,0 +1,55 @@
+from lightkube import Client
+from lightkube.core.exceptions import ApiError
+from lightkube.resources.core_v1 import Namespace
+
+from dss.config import DSS_NAMESPACE
+from dss.logger import setup_logger
+from dss.utils import does_namespace_exist, wait_for_namespace_to_be_deleted
+
+# Set up logger
+logger = setup_logger("logs/dss.log")
+
+
+def purge(lightkube_client: Client) -> None:
+    """
+    Removes all notebooks and DSS components.
+
+    Args:
+        lightkube_client (Client): The Kubernetes client.
+    """
+    try:
+        if not does_namespace_exist(lightkube_client, DSS_NAMESPACE):
+            logger.error(
+                f"Failed to purge DSS components. Namespace `{DSS_NAMESPACE}` does not exist."
+            )
+            logger.info(
+                "You might want to run\n"
+                "  dss status      to check the current status\n"
+                "  dss logs --all  to review all logs\n"
+            )
```

Comment:
@misohu 's input can be useful here

Reply:
I thought about it too but I think we should not suggest to `dss initialize` because if I'm a user and I'm trying to run `purge`, it sounds a bit awkward to suggest me that I should initialize.

Reply:
I agree with Noha maybe user accidentally run it twice and the recommendation will hint that maybe they need to reinitialize. I think we should add the dss initialize in the hint